readme.md

```diff
@@ -62,52 +62,39 @@ Pass `false` to not set `rel`s on links.
 > When using a `target`, add [`noopener` and `noreferrer`][mdn-a] to avoid
 > exploitation of the `window.opener` API.
 
+###### `options.protocols`
+
+Protocols used to filter against the [is-absolute](https://www.npmjs.com/package/is-absolute) package.
+(`Array.<string>`, default: `['http', 'https']`).
+
 ## Contribute
 
 See [`contributing.md` in `remarkjs/remark`][contributing] for ways to get
 started.
 
-This organisation has a [Code of Conduct][coc].  By interacting with this
+This organisation has a [Code of Conduct][coc]. By interacting with this
 repository, organisation, or community you agree to abide by its terms.
 
 ## License
 
 [MIT][license] © [Cédric Delpoux][author]
 
 [build-badge]: https://img.shields.io/travis/remarkjs/remark-external-links.svg
-
```

Comment:
I’m guessing prettier did this? Could you turn that off (`npm run test` should format the markdown the way this project likes it)

Reply:
Sure thing, pushed in my latest commit.